app/jobs/application_job.rb

```diff
@@ -6,4 +6,11 @@ class ApplicationJob < ActiveJob::Base
 
   # Most jobs are safe to ignore if the underlying records are no longer available
   # discard_on ActiveJob::DeserializationError
+  private
+
+  def log &block
+    puts "*" * 100
```

Comment:
@radinreth should write with human-readable "********** start/begin & finish/end ***********" s.th like this is more readable and easily to review back then just ****